src/Presence/kahoot.json

```
@@ -94,5 +94,97 @@
     "kahoot.of": {
         "message": "{0} of {1}",
         "description": "Used in the context of questions (i.e 1 of 3). {0} is the current question and {1} is the total amount of questions"
+    },
+    "kahoot.onPodium": {
+        "message": "On the Podium",
+        "description": "Used in the context of a player/team being withing the top 3"
+    },
+    "kahoot.pollAnswer": {
+        "message": "Answered Poll",
+        "description": "Used when a player is viewing the results of a poll"
+    },
+    "kahoot.drumRoll": {
+        "message": "Waiting for final ranking...",
+        "description": "Used when a player is waiting to receive their final ranking"
+    },
+    "kahoot.position": {
+        "message": "Position: {0}",
+        "description": "Used to display the position (ranking) of a player. For example, 'Position: 2'"
+    },
+    "kahoot.teamTalk": {
+        "message": "Discussing with team",
+        "description": "Used when a player is talking to their team during team talk time"
+    },
+    "kahoot.gameSummary": {
+        "message": "Looking at game summary",
+        "description": "Used when the host is viewing the summary of a game"
+    },
+    "kahoot.login": {
+        "message": "Logging in",
+        "description": "Used when a user is logging in"
+    },
+    "kahoot.createHome": {
+        "message": "Viewing the \"Create home\" page",
+        "description": "Used when the user is viewing the create home page"
+    },
+    "kahoot.discover": {
+        "message": "Viewing the \"Kahoot! Discover\" page",
+        "description": "Used when the user is viewing the discover page"
+    },
+    "kahoot.searchKahoots": {
+        "message": "Searching for Kahoots:",
+        "description": "Used when the user is searching for Kahoots. The search query will be displayed below."
+    },
+    "kahoot.kahootDetails": {
+        "message": "Viewing details of Kahoot:",
+        "description": "Used when the user is viewing the details of a Kahoot. The name of the Kahoot will be displayed below."
+    },
+    "kahoot.kahootProfile": {
+        "message": "Viewing Kahoot profile:",
+        "description": "Used when the user is viewing the profile of a Kahoot"
+    },
+    "kahoot.myKahoots": {
+        "message": "Viewing their Kahoots",
+        "description": "Used when the user is viewing their own kahoots"
+    },
+    "kahoot.userReports": {
+        "message": "Viewing game reports",
+        "description": "Used when the user is viewing their game reports"
```

consistency (assuming this is true)
```suggestion
        "message": "Viewing their game reports",
        "description": "Used when the user is viewing their own game reports"
```